docs/building.md

```diff
@@ -30,19 +30,17 @@ First clone the Keycloak repository:
     
 To build Keycloak run:
 
-    mvn clean install
+    mvn clean install -DskipTests=true
     
-This will build all modules and run the testsuite. 
+This will build all modules. The build of the Keycloak server is then available in the archive under the directory `quarkus/dist/target`.
 
-To build the ZIP distribution run:
+If you would also like to run testsuite during the build, you can omit the `-DskipTests=true` parameter. Please be aware that testsuite can take at least 3 hours to complete. However, there is an alternative if you want to avoid running the full testsuite on your laptop and at the same time ensure that your changes in the code did not cause regressions. To check if your changes will cause a regression, you should push the changes to a GitHub fork and verify if all the checks are successful. Those checks run testsuite and do other verifications.
```

Comment:
I'd probably leave this out, as it should rather be captured in the testing documentation

Reply:
Those were the changes suggested by @mposolda. I don't mind removing it if we agree.